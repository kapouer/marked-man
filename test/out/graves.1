TEST()                                                                  TEST()

NNAAMMEE
       tteesstt

tthhiiss ttiittllee wwiillll rreennddeerr ssttrraannggeellyy
       This is the body text.

                                 January 1979                           TEST()
