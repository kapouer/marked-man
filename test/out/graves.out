[4mTEST[24m(1)                     General Commands Manual                    [4mTEST[24m(1)

[1mNAME[0m
       [1mtest[0m

[1mthis title will render strangely[0m
       This is the body text.

                                 January 1979                          [4mTEST[24m(1)
